injector/FindLocalImports.py

```
@@ -0,0 +1,72 @@
+        pathsToCheck.append(path)
+    elif isinstance(node, ast.ImportFrom):
+        module = os.path.join(rootDir, node.module.replace('.','/'))
+        name = os.path.join(node.names[0].name.replace('.','/'))
+        pathsToCheck.append(f"{module}/{name}.py")
+        pathsToCheck.append(f"{module}.py")
+    
+    # Check for valid paths
+    validPaths = []
+    for path in pathsToCheck:
+        try:
+            open(path)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)